Sources/ParseSwift/Coding/ParseEncoder.swift

```diff
@@ -154,11 +154,11 @@ internal struct _ParseEncoderKeyedEncodingContainer<Key: CodingKey>: KeyedEncodi
     }
 
     mutating func superEncoder() -> Encoder {
```

Comment:
Are you not able yo mark these as unavailable?

Reply:
Nope. The compiler produces the following error:

```
Unavailable instance method 'superEncoder()' was used to satisfy a requirement of protocol 'UnkeyedEncodingContainer'
```